.github/workflows/cleanup-branch-builds.yml

```diff
@@ -9,6 +9,10 @@ jobs:
   delete-package:
     name: Delete Github Package for Branch
     runs-on: ubuntu-20.04
+    matrix:
+      package:
+        - org.liquibase.liquibase-core
+        - org.liquibase.liquibase-extension-testing
```

Comment:
Is deleting branch builds needed if we aren't publishing branch builds?

Reply:
Good point, removed!